script/ExecTransaction.s.sol

```diff
@@ -3,10 +3,14 @@ pragma solidity ^0.8.0;
 
 import {QuickSort} from "./libraries/QuickSort.sol";
 import {SafeTxDataBuilder, Enum} from "./SafeTxDataBuilder.sol";
+import {console2} from "forge-std/console2.sol";
 
 contract ExecTransaction is SafeTxDataBuilder {
     using QuickSort for address[];
 
+    // Each signature saved in data/signatures.txt is (2 bytes 0x prefix + 130 bytes data =) 132 bytes long and suffixed by 1 byte of newline character.
```

Comment:
I'm not sure why is it 130 bytes...

Reply:
Signature is 65 bytes:
1. 32 bytes of r
2. 32 bytes of s
3. 1 byte of v

But each byte of signature is a character utf8 hex-encoded into 2 bytes so each signature is 130 bytes of data

Reply:
> But each byte of signature is a character utf8 hex-encoded into 2 bytes so each signature is 130 bytes of data

Oh yeah that's why thx